recipes-extended/jailhouse/jailhouse_0.11.bb

```diff
@@ -0,0 +1,121 @@
+SUMMARY = "Jailhouse, i.MX fork"
```

Comment:
Do you maybe need/want to put `Copyright` and `SPDX-License-Identifier` in the recipe?

Reply:
I could set SPDX to MIT but it's not necessary.
I will not add copyright.

Reply:
Copyrights in recipes indicate who wrote the recipe, if originator didn't add a copyright on it, we should leave it without one.

Reply:
Also, it seem LICENSE should be change as well, it's not about a dual license, it's GPL-2.0

Reply:
OK, I see. I guess since you've originated this work from [this repository](https://bitbucket.org/retotech/meta-jailhouse) - maybe you can put this link in the recipe comments header?

In that case - the original author and copyright would be retained IMHO.

Reply:
Indeed, it seems debatable..
I think that reference should stay in the commit message, to have it in history rather than in the header, to always have it there.
Anyway, to be copyrightable, something must contain an original expression. We just added / adapted some things to the original to work with i.MX.